.gitignore

```diff
@@ -28,7 +28,7 @@ build/
 
 # for a library or gem, you might want to ignore these files since the code is
 # intended to run in multiple environments; otherwise, check them in:
-# Gemfile.lock
+Gemfile.lock
```

Comment:
Good question.  On this topic, I found a few references to http://yehudakatz.com/2010/12/16/clarifying-the-roles-of-the-gemspec-and-gemfile/  

I wasn't sure if it is still valid, however, people still seem to reference it.  The basic idea is that since gem is used for installing the library, the authoritative version of the dependencies are in  in the gemspec file not the Gemfile.lock.